user/storage.md

```diff
@@ -13,29 +13,46 @@ The following material was adapted from the
 [Pangeo Cloud User Guide](https://pangeo.io/cloud.html)
 :::
 
-## Your Home Directory
+## The JupyterHub Filesystem
 
 Your notebook server is a linux "virtual machine" with its own filesystem.
 Your are not on a shared server; you are on your own private server.
-Your username is ``jovyan``, and your home directory is ``/home/jovyan``.
-This is the same for all users.
-
-Your home directory is intended only for notebooks, analysis scripts, and small datasets (< 1 GB).
-It is not an appropriate place to store large datasets.
-No one else can see or access the files your home directory.
 
 The easiest way to move files in and out of your home directory is via the JupyterLab web interface.
 Drag a file into the file browser to upload, and right-click to download back out.
 You can also open a terminal via the JupyterLab launcher and use this to ssh / scp / ftp to remote systems.
 However, you can’t ssh in!
 
-## The `shared` Directory
+### Your Home Directory
+
+Your username is ``jovyan``, and your home directory is ``/home/jovyan``.
+This is the same for all users, but no one else can see or access the files *your* home directory.
+
+``/home/jovyan`` is a persistant network-attached drive. Any files you put there will be there when you
+log out and log back into the JupyterHub. 
+
+The ``/home/jovyan`` space is limited to 10 GB, which is a common
```

Comment:
I'm not sure how true this is for 2i2c hubs. Mileage may vary depending on how the NFS store is setup, and I don't think we operate per-user quotas (though we should).

Reply:
Even if it's not enforced, we've been telling people this limit exists for hackweeks/pangeo to try and limit ballooning storage :) but I can remove of reword if you prefer

Reply:
My understanding is that we don't have hard limits, but I'd be +1 on advertising a limit and telling people they can go beyond it if they ask

Reply:
+1 to @sgibson91 - I think we shouldn't state anything here about limits. I like that the home directory section states you shouldn't keep large files in here, and I think that's the right way to go - provide guidelines for what you can / can't do, rather than set an arbitrary limit that we can't enforce.

Reply:
Can we just say "we recommend storing no more than 10GB of data in your home directory"? I think people will benefit from having some kind of number in their mind even if it is a soft limit